### PR TITLE
Change the java version

### PR DESCRIPTION
**Pull request by SIGSTACKFAULT**
## Change list

on my machine, Maven defaults to 1.7, which doesn't work. add instructions to update the version in `pom.xml`.
 
## Types of changes

- [x] No changes in production code.
- [ ] Bugfix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

## Details

on my Maven (and I _think_ everyone's?), the default java.compiler.source and java.compiler.target are 1.7, which doesn't work on my machine.

### CODE REVIEW

**Review by asolntsev — APPROVED** [review]

File: docs/Installing-the-project.md [hunk 1]
```diff
@@ -17,6 +17,14 @@ Add the following to pom.xml:
 </dependency>
 ```
 
+If you haven't already, change the Java version:
+```
+<properties>
+    <maven.compiler.source>11</maven.compiler.source>
```
Comment (mykola-mokhnach) [inline docs/Installing-the-project.md:23]: the minimum supported one is actually 1.8

Comment (SIGSTACKFAULT) [reply docs/Installing-the-project.md:23]: alright, i'll change it to 1.8

**Comment by mykola-mokhnach** [comment]
Thanks, could you please sign the CLA?

**Comment by SIGSTACKFAULT** [comment]
@mykola-mokhnach i'm waiting for LFX to confirm what they need my home address for.

**Comment by SIGSTACKFAULT** [comment]
@mykola-mokhnach signed!

**Pull request merged**